sapi_manifests/cmon/template

```diff
@@ -22,6 +22,9 @@
         "subResources": ["alias","destroyed","server_uuid","state"],
         "instance": "{{{INSTANCE_UUID}}}"
     },
+    "server_opts" : {
+        "http_accept_encoding": "{{HTTP_ACCEPT_ENCODING}}"
```

Comment:
I think this is the first real SAPI tunable for CMON? If so, it would likely be helpful to operators to start a "SAPI Configuration" section in docs/README.md (??) a la CNAPI's https://github.com/joyent/sdc-cnapi/blob/master/docs/index.md#sapi-configuration or VMAPI's https://github.com/joyent/sdc-vmapi/blob/master/docs/index.md#sapi-configuration

Reply:
This could totally be in a separate commit if this (already approved) change wants to get in now.

Reply:
Will do in a separate commit. I want to get this in so that QA team gets a chance to test it.